src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/HttpApi/ApiClient.cs

```diff
@@ -199,9 +199,9 @@ private async Task<ResponseStreamHolder> CaptureDumpAsync(string processKey, Dum
         /// <summary>
         /// GET /logs/{pid}?level={logLevel}&durationSeconds={duration}
         /// </summary>
-        public Task<ResponseStreamHolder> CaptureLogsAsync(int pid, TimeSpan duration, LogLevel? logLevel, CancellationToken token)
+        public Task<ResponseStreamHolder> CaptureLogsAsync(int pid, TimeSpan duration, LogLevel? logLevel, CancellationToken token, string contentType = ContentTypes.ApplicationNDJson)
```

Comment:
Couple of things here:
- Async methods typically place the `CancellationToken` parameter as the last parameter of the method.
- Don't give the `contentType` parameter a default value. Have the tests explicitly pass in the argument.
- Let's change the `contentType` parameter to `LogFormat logFormat` so that the conversion from the log format to the content type is kept internal to the `ApiClient`, thus it's not a concern of each test.